lib/msf/core/post/file.rb

```diff
@@ -638,6 +642,153 @@ def copy_file(src_file, dst_file)
+
+      # Grab info on the groups the current user belongs to so that we can do our comparisons.
+      # The following output shows what this might look like, showing that we get a group name
+      # as a string, then the type of the SID, the SID itself, followed by its attributes.
+      #
+      # Keep in mind these SID's are all SIDs that the current user is a member of.
+      #
+      #
+      # C:\Users>whoami /groups
+      # GROUP INFORMATION
+      # -----------------
+      #
+      # Group Name                                 Type             SID                                            Attributes
+      #
+      # ========================================== ================ ============================================== ==================================================
+      # Everyone                                   Well-known group S-1-1-0                                        Mandatory group, Enabled by default, Enabled group
+      # BUILTIN\Users                              Alias            S-1-5-32-545                                   Mandatory group, Enabled by default, Enabled group
+      # BUILTIN\Performance Log Users              Alias            S-1-5-32-559                                   Mandatory group, Enabled by default, Enabled group
+      # BUILTIN\Administrators                     Alias            S-1-5-32-544                                   Group used for deny only
+      # NT AUTHORITY\INTERACTIVE                   Well-known group S-1-5-4                                        Mandatory group, Enabled by default, Enabled group
+      # CONSOLE LOGON                              Well-known group S-1-2-1                                        Mandatory group, Enabled by default, Enabled group
+      # NT AUTHORITY\Authenticated Users           Well-known group S-1-5-11                                       Mandatory group, Enabled by default, Enabled group
+      # NT AUTHORITY\This Organization             Well-known group S-1-5-15                                       Mandatory group, Enabled by default, Enabled group
+      # LOCAL                                      Well-known group S-1-2-0                                        Mandatory group, Enabled by default, Enabled group
+      #                                            Unknown SID type S-1-5-21-3070936213-306261907-1348773959-36273 Mandatory group, Enabled by default, Enabled group
+      #                                            Unknown SID type S-1-5-21-3070936213-306261907-1348773959-45105 Mandatory group, Enabled by default, Enabled group
+      # **** CUT FOR BREVITY ***
+      # Authentication authority asserted identity Well-known group S-1-18-1                                       Mandatory group, Enabled by default, Enabled group
+      # Mandatory Label\Medium Mandatory Level     Label            S-1-16-8192
+      #
+      # C:\Users>
```

Comment:
make this part of rdoc for `win_find_current_user_groups`

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/15987/commits/9aee8e2b4264b96da724aa2f2606851d0cfe52b5